resources/report-table-formatter.js

```diff
@@ -0,0 +1,85 @@
+{
+    "pangolin strain (#SNPs)": function format(value) {
+        $(function () {
+            $('[data-toggle="tooltip"]').tooltip()
+        })
+
+        if (value !== "no strain called") {
+            var lineage = value.split(' ')[0];
+            var link = "<a data-toggle=\"tooltip\" data-placement=\"top\" title=\"Linkout to cov-lineages\" href='https://cov-lineages.org/lineages/lineage_" + lineage + ".html' target='_blank'>" + lineage + "</a>";
+            return link + value.split(lineage).pop();
+        } else {
+            return value;3
+        }
+    },
+    "variants of interest": function format(value) {
+        $(function () {
+            $('[data-toggle="popover"]').popover()
+        })
+
+        $(function () {
+            $('[data-toggle="tooltip"]').tooltip()
+        })
+
+        $('.popover-dismiss').popover({
+            trigger: 'focus'
+        })
+
+        var variants = value.split(' ');
+        var genes = [];
+        var vafs = {};
+        var vats = {};
+        for (var v of variants) {
+            var split = v.split(':');
+            genes.push(split[0]);
+        }
+
+        for (var g of genes) {
+            vafs[g] = [];
+            vats[g] = [];
+        }
+
+        for (var v of variants) {
+            var split = v.split(':');
+            vafs[split[0]].push(split[1]);
+            vats[split[0]].push(split[2]);
+        }
+
+        const table = "<table class=\"table\">\n" +
+            "  <thead>\n" +
+            "    <tr>\n" +
+            "      <th scope=\"col\">Variant</th>\n" +
+            "      <th scope=\"col\">VAF</th>\n" +
+            "    </tr>\n" +
+            "  </thead>" +
+            "<tbody>";
+
+        const table_end = "</tbody></table>";
+
+        var tables = {};
+
+        for (g of genes) {
+            var body = "";
+            for (let i = 0; i < vafs[g].length; i++) {
+                var row = "    <tr><td scope=\"col\">" + vafs[g][i] + "</td><td>"  + vats[g][i] + "</td></tr>";
```

Comment:
Use the format string syntax:
```
`<tr>...${vafs[g][i]}...`
```

Reply:
And use that everywhere in this file where strings are currently concatenated to fill in values from variables.

Reply:
Changed to format strings in 5d29c4a30442d823e81a14a7c02c42daa41a5699.